app/assets/stylesheets/_commons.css.scss

```diff
@@ -174,6 +161,10 @@ body {
     background: rgb(171, 171, 171);
     padding: 17px;
     border-radius: 7px;
+    margin-top: 0.5rem;
+    position: relative;
+    background: white;
+    border: 2px solid #DDDDDD;
```

Comment:
Color literals like `#DDDDDD` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#DDDDDD` should be written as `#dddddd`